js/menu.js

```diff
@@ -50,6 +50,17 @@ function init() {
         'right': '15px'
       });
     }
+
+    $('[data-fl-exit-app]').html([
+      '<div class="fl-bottom-bar-icon-holder">',
```

Comment:
Take a look at https://github.com/Fliplet/fliplet-menu-bottom-bar/blob/master/js/menu.js#L10-L64

1. It uses the `addExitAppMenuLink` hook to customise **how** the exit app menu link is added. What you're doing here is assessing the page for exit app menu links after it's already added, which is not the preferred way as it relies on a specific order of rendering, which is subject to change.
1. It uses the `fl-bottom-bar-icon-holder` class because it's specific to the style of the bottom menu bar. It doesn't make sense why you're using the `fl-bottom-bar-icon-holder` class for the circle menu. Have a loot at how the circle menu specifically [structures its DOM](https://github.com/Fliplet/fliplet-menu-circle/blob/master/build.html#L28-L35) for the menu. 
1. It returns a rejected promise at the end such that the default exit app menu link render process isn't executed.

Reply:
@tonytlwu altered the solution to use the hook, also aligned icon so it would be in the same line as other icons. 
![screenshot-2020-03-19_08 41 25 024](https://user-images.githubusercontent.com/53430352/77039142-0235bb80-69be-11ea-862c-089458c4b285.png)